### YamlMime:UniversalReference
api_name: []
items:
- children: []
  class: azure.iot.device.exceptions.ClientError
  fullName: azure.iot.device.exceptions.ClientError
  inheritance:
  - inheritance:
    - inheritance:
      - inheritance:
        - type: builtins.object
        type: builtins.BaseException
      type: builtins.Exception
    type: azure.iot.device.common.chainable_exception.ChainableException
  langs:
  - python
  module: azure.iot.device.exceptions
  name: ClientError
  source:
    id: ClientError
    path: azure-iot-device\azure\iot\device\exceptions.py
    remote:
      branch: master
      path: azure-iot-device\azure\iot\device\exceptions.py
      repo: https://github.com/Azure/azure-iot-sdk-python
    startLine: 26
  summary: Generic error for a client
  syntax:
    content: ClientError(message=None, cause=None)
  type: class
  uid: azure.iot.device.exceptions.ClientError
references: []
